core/src/main/java/org/bitcoinj/core/Peer.java

```diff
@@ -578,7 +580,7 @@ protected void processMessage(Message m) throws Exception {
             processUTXOMessage((UTXOsMessage) m);
         } else if (m instanceof RejectMessage) {
             log.error("{} {}: Received {}", this, getPeerVersionMessage().subVer, m);
-        } else if(m instanceof DarkSendQueue) {
+        } else if(m instanceof CoinJoinQueue) {
             //do nothing
```

Comment:
`DarkSendQueue` is replaced with `CoinJoinQueue`, but do we still need to do nothing here?

Reply:
For now we can do nothing.  The goal with this PR is to add the message classes.  We are not sending or receiving or processing any of the messages.